File: Scraper/Vimar/models/product.py
class Product:
    def __init__(self, id: str, name: str, description: str, etim: str):
        self.__id = id
        self.__name = name
        self.__description = description
        self.__etim = etim
    
    def get_id(self) -> str:
        return self.__id
    
    def get_name(self) -> str:
        return self.__name
    
    def get_description(self) -> str:
        return self.__description
    
    def get_etim(self) -> str:
        return self.__etim

File: docker/volumes/functions/models/product.ts
export class Product {
    constructor(
        private id: string,
        private name: string,
        private description: string,
        private etim: string
    ) {}

    public getID(): string {
        return this.id;
    }

    public getName(): string {
        return this.name;
    }

    public getDescription(): string {
        return this.description;
    }

    public getEtim(): string {
        // function to copy from github
        return this.etim;
    }

    public setName(name: string): void {
        this.name = name;
    }

    public setDescription(description: string): void {
        this.description = description;
    }

}